dotnet/packages/Microsoft.Bot.Builder.M365/Microsoft.Bot.Builder.M365/AI/AzureContentSafety/AzureContentSafetyClient.cs

```diff
@@ -53,7 +53,7 @@ public virtual async Task<AzureContentSafetyTextAnalysisResponse> ExecuteTextMod
                 string responseJson = await httpResponse.Content.ReadAsStringAsync();
                 AzureContentSafetyTextAnalysisResponse result = JsonSerializer.Deserialize<AzureContentSafetyTextAnalysisResponse>(responseJson) ?? throw new SerializationException($"Failed to deserialize moderation result response json: {content}");
 
-                Verify.NotNull(result.HateResult);
+                Verify.ParamNotNull(result.HateResult);
```

Comment:
Oh, my bad... I added this line for local test but forgot removing it from the commit.
It would be great if you could help remove this line by the way in this PR. Or I could created a PR to do that.